Editor/Resources/EditorWindow/Components/DeploymentParameters.uxml

```diff
@@ -0,0 +1,36 @@
+﻿<!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<ui:UXML xmlns:ui="UnityEngine.UIElements">
+    <Style src="../common.uss"/>
+    <ui:VisualElement name="FleetName" class="form-row">
+        <ui:Label name="ManagedEC2ParametersFleetNameLabel" class="form-row__label" text="Fleet Name"/>
+        <ui:TextField name="EC2FleetNameInput" class="form-row__input text-input" value="GameName-ManagedFleet"/>
```

Comment:
* Please be consistent with naming. Let's use `ManagedEC2` in all spots rather than `EC2`.
* This file is called `DeploymentParameters` but only seems to be specific to the Managed Fleet scenario. Can the file be renamed as `ManagedEC2DeploymentParamaters` or something more specific?
* The parent elemt is just named `FleetName`, should it be `ManagedEc2FleetName` instead?
* Please make sure these recommendations are applied to all occurrences in this file.

Reply:
I would pick one convention and make sure all spots follow the same. Please review this file and `DeploymentScenarios.uxml` for consistency.

Reply:
If all scenarios are specific to ManagedFleets, I would consider dropping the `ManagedEC2` prefix just to make things more concise and readable.

Reply:
I have stuck to the more verbose naming as it will match how the elements get translations applied